x-pack/test/security_solution_endpoint/page_objects/policy_page.ts

```diff
@@ -101,5 +101,13 @@ export function EndpointPolicyPageProvider({ getService, getPageObjects }: FtrPr
     async findDatasourceEndpointCustomConfiguration(onEditPage: boolean = false) {
       return await testSubjects.find(`endpointDatasourceConfig_${onEditPage ? 'edit' : 'create'}`);
     },
+
+    /**
+     * Finds and returns the onboarding button displayed in empty List pages
+     */
+    async findEmptyStateButton() {
```

Comment:
rename suggestion: `findOnboardingStartButton()`.

`*EmptyStatusButton` seems to generic to me because it king of matches the Eui component name